### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by Logan9312**
Adding a codeowners file to auto-assign us to review Pull Requests

### CODE REVIEW

**Review by Sundance636 — APPROVED** [review]

**Review by rustyRustacean — APPROVED** [review]

**Comment by Nandan-18** [comment]
updated rulesets to require reviews from codeowners

**Pull request merged**
